scripts/createImage.sh

```diff
@@ -0,0 +1,42 @@
+#/bin/bash
+set -e
+
+# generate armada docker image
+#  run like so: scripts/createImage.sh /home/gbj/incoming/spark testing
+SPARK_ROOT=$1
+IMAGE_NAME=$2
+
+# Get the dependencies to be copied into docker image
+mvn --batch-mode package dependency:copy-dependencies
+dependencies=(
+    target/armada-cluster-manager-1.0.0-SNAPSHOT.jar
+    target/dependency/lenses_2.13-0.11.13.jar
+    target/dependency/scalapb-runtime_2.13-0.11.13.jar
+    target/dependency/scalapb-runtime-grpc_2.13-0.11.13.jar
+    target/dependency/scala-armada-client_2.13-0.1.0-SNAPSHOT.jar
+)
+
+# Copy dependencies to the docker image directory
+for d in ${dependencies[@]}; do
+    cp $d $SPARK_ROOT/assembly/target/scala-2.13/jars
+done
```

Comment:
The `"${dependencies[@]}"` should expanded by bash to the individual files in `dependencies`, wrapped by quotes:
```suggestion
cp "${dependencies[@]}" $SPARK_ROOT/assembly/target/scala-2.13/jars/
```